lib/core/deviceInfo.js

```diff
@@ -37,5 +37,6 @@ module.exports = {
                                  'Blood glucose and ketone meter'),
   'InsuletOmniPod': infoBuilder('Insulet OmniPod', 'Upload .ibf file from PDM.'),
   'OneTouchUltra2': infoBuilder('OneTouch Ultra2', ''),
-  'BayerContourNext': infoBuilder('Bayer Contour Next', '')
+  'BayerContourNext': infoBuilder('Bayer Contour Next', 'Blood glucose meter'),
```

Comment:
I'll probably take out the 'Blood glucose meter'...we've only been adding a "subtitle" if there's something unusual about the device, like it reading ketones or whatever.